### PR TITLE
zen changes to snap kick, and set ammo count

### LINKED ISSUE #102: Zenyatta changes
Remove Snap Kick passive
Max ammo reduced from 25 -> 20

### PR DESCRIPTION
**Pull request by snappycreeper**
fixes #102

### CODE REVIEW

File: src/constants/ow1_constants.opy [hunk 1]
```diff
@@ -165,6 +165,9 @@
 #!define OW1_WRECKING_BALL_HEALTH 500
 #!define OW1_WRECKING_BALL_ARMOR 100
 
+# Zenyatta
```
Comment (MaxwellJung) [inline src/constants/ow1_constants.opy:168]: Can you move zenyatta after zarya?

File: src/ow1/heroes/heroes.opy [hunk 2]
```diff
@@ -26,4 +26,6 @@
 #!include "ow1/heroes/widowmaker.opy"
 #!include "ow1/heroes/winston.opy"
 #!include "ow1/heroes/wreckingball.opy"
+#!include "ow1/heroes/zenyatta.opy"
```
Comment (MaxwellJung) [inline src/ow1/heroes/heroes.opy:29]: move zenyatta after zarya

**Review by MaxwellJung — CHANGES_REQUESTED** [review]
Other than the name ordering, everything looks good, as long as the 50% reduction is correct.

**Comment by snappycreeper** [comment]
Changed names and fixed the 50% knockback

**Review by MaxwellJung — APPROVED** [review]
Looks good!

**Pull request merged**